awx/sso/views.py

```diff
@@ -44,7 +44,7 @@ def dispatch(self, request, *args, **kwargs):
             logger.info(smart_text(u"User {} logged in".format(self.request.user.username)))
             response.set_cookie('userLoggedIn', 'true')
             current_user = UserSerializer(self.request.user)
-            current_user = JSONRenderer().render(current_user.data)
+            current_user = smart_text(JSONRenderer().render(current_user.data))
```

Comment:
could we do this here as well? https://github.com/rooftopcellist/awx/blob/devel/awx/api/generics.py#L93

Reply:
Done!